drafts/use-cases/1.entry-point.md

```diff
@@ -0,0 +1,77 @@
+## Entry point
+
+### Story
+As an application user
+I want my application to consume an entry point for the Web API by providing a base URL
+So I can start managing my events.
+
+### Usage
+```javascript
+var client = new HydraClient();
+var entryPoint = client.get("http://example.com/").getEntryPoint();
```

Comment:
This `client.get()` looks pretty weird as synchronous method, how about using promises in similar way to https://fetch.spec.whatwg.org/ ?

```js
var client = new HydraClient()
client.get("http://example.com/")
  .then(response => response.getEntryPoint())
  .then(entryPoint => {
    for (var operation of entryPoint.operations) {
      // do something with the *operation*
    }
  }).catch(err => { /* handle errors */ })
```